packages/explorer/README.md

```diff
@@ -1,85 +1,53 @@
 # World Explorer
 
 World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+For the full information about it, [see the docs](http://mud.dev/world-explorer).
 
-## Getting started
+## Installation
```

Comment:
I see why you would not include creation from template guidelines here but perhaps it would still make sense to add a link at least to MUD templates setup docs?

Reply:
Yes.